pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -12,6 +12,7 @@
 , proprietaryCodecs ? false, vivaldi-ffmpeg-codecs ? null
 , enableWidevine ? false, vivaldi-widevine ? null
 , commandLineArgs ? ""
+, pulseSupport ? config.pulseaudio or stdenv.isLinux, libpulseaudio ? null
```

Comment:
Please remove config.pulseaudio. Pulse does only work on Linux, so this would always be true. Also the `? null` part can br removed.